@xen-orchestra/web/src/components/hosts/hostsTable.vue

```diff
@@ -0,0 +1,253 @@
+<template>
+  <div class="hosts-table">
+    <UiTitle>{{ t('hosts') }}</UiTitle>
+    <div class="container">
+      <div class="table-actions">
+        <UiQuerySearchBar @search="value => (searchQuery = value)" />
+        <UiTopBottomTable :selected-items="0" :total-items="0">
+          <UiTablePagination v-bind="paginationBindings" />
+        </UiTopBottomTable>
+      </div>
+      <VtsDataTable
+        :no-data-message="hosts.length === 0 ? t('no-host-detected') : undefined"
+        :has-error
+        :is-ready="!tableReady"
```

Comment:
```suggestion
        :is-ready"
```
Actually this is loading indefinitely